docker/README.md

```diff
@@ -0,0 +1,149 @@
+
+```bash
+sudo apt install -y podman
+alias docker=podman
+```
+
+## Build a local container image
+
+Build a local container image `kubestellar` for the host architecture using the latest release of **KubeStellar** with the command:
+
+```bash
+make build
+```
+
+A desired version of **KubeStellar** can be specified by using the optional `KUBESTELLAR_VERSION` argument, such as `KUBESTELLAR_VERSION=v0.3.1`.
+
+The **KubeStellar** version is used to tag the container image, as shown below:
+
+```bash
+$ docker images
+REPOSITORY    TAG       IMAGE ID       CREATED          SIZE
+kubestellar   v0.3.1    0425aee2cda6   8 minutes ago    617MB
+kubestellar   v0.3.0    76d714261c6e   14 minutes ago   617MB
+```
+
+## Build and push a multi-architecture image
+
+Note that `docker buildx` is required for this option.
+
+Make sure to login into the registry with the desired user:
+
+```bash
+docker login -u <user> <registry>
+```
+
+Build and push a multi-architecture image using the command:
+
+```bash
+make buildx IMG=<registry>/<organization>/<image>
+```
+
+A desired version of **KubeStellar** can be specified by using the optional `KUBESTELLAR_VERSION` argument, such as `KUBESTELLAR_VERSION=v0.3.1`, Default value: the latest release version of **KubeStellar**.
+
+The list of desired target platforms can be specified by using the optional `PLATFORMS` argument. Default value; `PLATFORMS=linux/amd64,linux/arm64,linux/ppc64le`.
+
+## Run the container and access KubeStellar from the host OS
+
+Spin up a local or remote container image with the command:
+
+```bash
+make run
+export KUBECONFIG=${HOME}/.kcp/admin.kubeconfig
+export PATH=$PATH:${HOME}/kcp:${HOME}/kubestellar/bin
+```
+
+The following arguments can be used:
+
+- `IMG=[<registry>/<organization>/]<image>` select the container image, the default vakue is the local `kubestellar` image
+- `KUBESTELLAR_VERSION=[<version>|latest]`, select the image tag, the default value is the latest version of **KubeStellar**
+- `NAME=<container-name>` sets the name of the container, the default value is `kubestellar`
+
+The command above will
+
+- make port `6443` accessibe on the host OS
+- share a `~/.kcp` folder containing the `KUBECONFIG` file
+- create a `~/kcp` folder containing a copy of **kcp** plugins for the host OS architecture
+- create a `~/kubestellar` folder containing a copy of **KubeStellar** binaries for the host OS architecture
+
+At this point, we can check basic **KubeStellar** functionality with the command:
+
+```bash
+kubectl ws root
+kubectl ws tree
+```
+
+which should return something like:
+
+```text
+.
+└── root
+    ├── compute
+    └── espw
+```
+
+At this point, one can follow the instructions and examples in **KubeStellar** [Quickstart](https://docs.kubestellar.io/release-0.3/Getting-Started/quickstart/).
+
+## Checking the container logs
+
+The running container logs can be easily displayed using the command:
+
+```bash
+make logs

```

Comment:
That just does `docker logs` on the container, but the interesting logs are in files in the container.

Reply:
I have added the `kubestellar-logs` volume